src/Tizen.NUI.Components/Controls/SelectButton.cs

```diff
@@ -44,6 +44,8 @@ public class SelectButton : Button
 
         private SelectButtonAttributes selectButtonAttributes;
 
+        private Extents checkImagePadding = null;
```

Comment:
There is seperated CheckBox control. This class name is SelectButton. CheckBox & Radio Button are inherited from SelectButton.  checkImage is not suitable for Radio button. Why don't you use selectImage instead of checkImage?

Reply:
I've modified it.